src/_dialog-placeholders.scss

```diff
@@ -26,9 +26,7 @@ $fd-dialog-overlay-color: rgba(0, 0, 0, 0.6); // Should be fade(var(--sapBlockLa
   flex-direction: column;
   outline: none;
   position: absolute;
-  min-width: $fd-dialog-content-min-width;
```

Comment:
Why this has been removed?
![image](https://user-images.githubusercontent.com/17496353/93880499-de8d5e00-fcdd-11ea-8baa-57db09ad747b.png)

Reply:
This was part of my attempt to put sizing in to the Dialog because I thought that the min-width/max-width specified in the Dialog page sizing section were actually supposed to be set by the Dialog. It's unfortunate there's no actual sizing, since the iframe does not have dimensions of its own and simply fills its container.

I reverted any changes to the dialog and set the width inside the story manually so now both height and width of the iframe will be up to the consumer.